spokestack/wakeword/tflite.py

```diff
@@ -59,17 +66,18 @@ def __init__(
         )
         self.encode_width: int = wake_encode_width
         self.state_width: int = wake_encode_width
+        self._hann_window = np.hanning(self.window_size)
 
         self.sample_window: RingBuffer = RingBuffer(shape=[self.window_size])
-        self.fft_window: RingBuffer = RingBuffer(
-            shape=[1, int(self.window_size / 2 + 1)]
-        )
+        self.fft_frame = np.empty((1, int(self.window_size / 2 + 1)), np.float32)
```

Comment:
you shouldn't need a member for this, just pass the frame from the `_analyze` method to the `_filter` method when you call it